From 899e7ee44b41485ea2c7dcba31db43db9dd5fb08 Mon Sep 17 00:00:00 2001
From: Alec Mouri <alecmouri@google.com>
Date: Fri, 26 Oct 2018 15:49:45 -0700
Subject: [PATCH 20/20] Respect source crop when capturing layers.

Source crop we receive from window manager appears to be correct, no
need for this workaround.

Bug: 117892959
Test: adb shell screencap, rotation animation, power cycle display
Change-Id: I54ea3fb561ce4f2ea6f48079ad51efdd9984d186
Merged-In: I54ea3fb561ce4f2ea6f48079ad51efdd9984d186
---
 services/surfaceflinger/SurfaceFlinger.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/services/surfaceflinger/SurfaceFlinger.cpp b/services/surfaceflinger/SurfaceFlinger.cpp
index fcc4dfe0b..e7812ee90 100644
--- a/services/surfaceflinger/SurfaceFlinger.cpp
+++ b/services/surfaceflinger/SurfaceFlinger.cpp
@@ -4890,10 +4890,6 @@ status_t SurfaceFlinger::captureScreen(const sp<IBinder>& display, sp<GraphicBuf
         device = getDisplayDeviceLocked(display);
         if (!device) return BAD_VALUE;
 
-        // ignore sourceCrop (i.e., use the projected logical display
-        // viewport) until the framework is fixed
-        sourceCrop.clear();
-
         // set the requested width/height to the logical display viewport size
         // by default
         if (reqWidth == 0 || reqHeight == 0) {
-- 
2.17.1

